src/main/java/edu/cornell/kfs/module/cg/document/validation/impl/AwardExtensionRule.java

```diff
@@ -91,6 +92,29 @@ public void setupBaseConvenienceObjects(MaintenanceDocument document) {
         super.setNewBo(newAwardCopy);
     }
 	
+    /**
+     * checks if the required federal pass through agency number is filled in if the
+     * federal pass through indicator is yes
+     *
+     * @return success

```

Comment:
The "return success" text is a little confusing without looking at the method contents. I would suggest either removing this line of the comment, or providing a more detailed explanation on what values are returned when.